diff --git a/Client.js b/Client.js
index 90e65d1cef57caf056d2e4a1f9e8ca7b17f3f4ec..221602bf98cc1532d563ae7d21554dd847517bfd 100644
--- a/Client.js
+++ b/Client.js
@@ -17,7 +17,7 @@ class Client extends events_1.EventEmitter {
     constructor(opts = {}) {
         super();
         this.reconnectAttempts = 0;
-        this.setMaxListeners(100);
+        //this.setMaxListeners(100);
         // Some EventEmitter shims don't include off()
         this.off = this.removeListener;
         this.updateConfig(opts);
